plugins/legacyoutput.md

```diff
@@ -8,7 +8,9 @@ keywords: legacyoutput filter content cms email flash
 
 The `legacyoutput` plugin changes TinyMCE's output, producing legacy elements such as `font`, `b`, `i`, `u`, `strike`, and use `align` attributes.
 
-This plugin can be useful if you want to use TinyMCE in an HTML mail client or to render contents to Flash.
+> **Important**: This plugin changes the editor content to HTML3. Using this plugin will greatly reduce the functionality of the editor.
+
+This plugin can be useful if you want to use {{site.productname}} in an HTML mail client or to render contents to Flash.
```

Comment:
Also, I don't think this part is accurate anymore. Maybe it should say `legacy HTML mail clients`?

Reply:
Fixed?